test/blocks/index.js

```diff
@@ -0,0 +1,73 @@
+import mocha from 'mocha';
+import chai, { expect } from 'chai';
+import dirtyChai from 'dirty-chai';
+
+import * as blocks from '@wordpress/blocks';
+
+chai.use( dirtyChai );
+
+describe( 'blocks API', () => {
+	// TODO: We probably want a way to undo this, and split this logic out into
```

Comment:
Would `blocks.unregisterBlock` be a reasonable reasonable to work around this problem?